packages/plugins/users-permissions/admin/src/hooks/usePlugins/index.js

```diff
@@ -18,10 +18,11 @@ const usePlugins = (shouldFetchData = true) => {
       dispatch({
         type: 'GET_DATA',
       });
+      const { get: getClient } = getFetchClient();
```

Comment:
Mh, so we have a naming conflict the we will see quite often 🤔 What do you think about `getRequest` instead? `get` is not really a client.

Reply:
What do you think about not destructuring instead of renaming the variable?
```
fetchClient = getFetchClient()

fetchClient.get()
```

Reply:
ok

Reply:
done